ros2_batch_job/__main__.py

```diff
@@ -380,6 +380,14 @@ def build_and_test(args, job):
     # xunit2 format is needed to make Jenkins xunit plugin 2.x happy
     with open('pytest.ini', 'w') as ini_file:
         ini_file.write('[pytest]\njunit_family=xunit2')
+    # check if packages have a pytest.ini file and add the xunit2
+    # format if it is not present
+    from pathlib import Path
+    for path in Path('.').rglob('pytest.ini'):
+        with open(path, "r+") as pytest:
+            if 'xunit2' not in pytest.read():
```

Comment:
This check if fragile. The string could be only in a comment.

Reply:
changed it in a6ce403 to use `ConfigParser`